drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_ops.c

```diff
@@ -224,21 +224,18 @@ static void hyper_dmabuf_ops_release(struct dma_buf *dma_buf)
 	dev_dbg(hy_drv_priv->dev, "%s: clear imported->dma_buf\n", __func__);
 	imported->dma_buf = NULL;
 
-	imported->importers--;
+	BUG_ON(imported->importers != 1);
```

Comment:
Thanks for the patch. Will test it.  
I guest  it might get panic  here probably.